src/ER/hospitalinfo.css

```diff
@@ -0,0 +1,97 @@
+.DiversionStyle{
```

Comment:
Since these styles will be used in the Beds.js Beds component, let's call this Beds.scss per the comment above, and let's start renaming the classes to use BEM naming... beds__component, beds__othercomponent, etc...